src/yadism/coefficient_functions/light/f2_nc.py

```diff
@@ -1,3 +1,4 @@
+from .. import partonic_channel as epc
```

Comment:
I actually think we can avoid adding all this empty valence channels and this will never shows up in f2. 
Do you agree? 
Same for gluon and singlet in PV structure functions.

Reply:
you can change if `kernels` does not access such a class (which it typically does in a observable independent matter i.e. `f2_nc` is dynamic)

Reply:
I disagree, now that we are complicating the structure and we have `is_pv` around, be sure not ending up in `f3.Gluon` would be an nice check.